src/utils.jl

```diff
@@ -95,31 +95,14 @@ function to_extent(edges::Vector{Edge})
     Extents.Extent(X=x, Y=y)
 end
 
-function to_points(x, ::Type{T} = Float64) where T
-    points = Vector{TuplePoint{T}}(undef, _npoint(x))
-    _to_points!(points, x, 1)
-    return points
-end
+_to_point3(T, p) = (T(GI.x(p)), T(GI.y(p)), T(GI.z(p)))
```

Comment:
I think we should make these more official...

like `xy` and `xyz` without underscores, so we can use them everywhere.